libtransmission/net.h

```diff
@@ -147,22 +148,35 @@ enum tr_address_type : uint8_t
     NUM_TR_AF_INET_TYPES
 };
 
-std::string_view tr_ip_protocol_sv(tr_address_type type) noexcept;
+auto constexpr inline IpProtocolToSv = std::array{ std::string_view{ "IPv4" }, std::string_view{ "IPv6" } };
+static_assert(std::size(IpProtocolToSv) == NUM_TR_AF_INET_TYPES);
 
-struct tr_socket_address;
+auto constexpr inline IpProtocolToAf = std::array{ AF_INET, AF_INET6 };
+static_assert(std::size(IpProtocolToAf) == NUM_TR_AF_INET_TYPES);
```

Comment:
Same question here about new items in the global namespace without a `tr_` prefix.

Reply:
f58a16a